tests/fixtures/subscription_sub_fakefakefakefakefake0001.json

```diff
@@ -5,7 +5,7 @@
     "billing": "charge_automatically",
     "billing_cycle_anchor": 1557995176,
     "billing_thresholds": null,
-    "cancel_at": null,
+    "cancel_at": 1624553655,
```

Comment:
Hi, one note on this - these JSON fixture files are generated (using `regenerate_test_fixtures`) so shouldn't be updated by hand (otherwise the change will be lost next time we run the script).

In this case to test the field the simplest change is probably to update the value of `subscription_fake` within the test before the call to `sync_from_stripe_data`?